tools/ja-rule/ja-rule-controller.cpp

```diff
@@ -102,7 +121,6 @@ class ControllerImpl : public ola::rdm::DiscoveryTargetInterface,
     RDMCommandSerializer::Pack(*request, data, &rdm_length);
     m_device->SendMessage(OpenLightingDevice::RDM_REQUEST, data, rdm_length);
 
-    // TODO(simon): do we need to check there isn't already one in-flight?
```

Comment:
The next step is to wrap this in a QueuingRDMController, which enforces serialization and also handles ACK overflows.